record-and-playback/video/playback/video/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title><%= meetingName %></title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="video-js/video-js.min.css">
+    <link rel="stylesheet" href="css/video.css">
+    <script src="video-js/video.min.js"></script>
+    <script src="js/popcorn-complete.min.js"></script>
+    <script src="js/popcorn.chattimeline.js"></script>
+    <script src="js/video.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <header>
+      <h1><%= meetingName %></h1>
+    </header>
+
+    <main>
+      <div id="main-area">
+        <div id="error-area">
+          <p id="firefox-format-warning" style="display:none">Firefox on your operating system does not currently support the video format used in this recording. Please try a different browser, such as Safari or Chrome.</p>
+        </div>
+        <div id="video-area">
+          <video id="video" class="video-js vjs-fill vjs-default-skin vjs-big-play-centered" controls>
+            <% video_props['formats'].each_with_index do |format, i| %>
+              <source src="capture-<%= i %>.<%= format[:extension] %>" type="<%= CGI.escapeHTML(format[:mimetype]) %>">
```

Comment:
Please rename the file creating to `video` instead of `capture`, as we discussed!

Reply:
Done; the index template should now be loading the correct video filename.